articles/integrations/marketing/adobe-campaign/index.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
Is there a reason why you created separate folders for this? You could name your doc `articles/integrations/marketing/adobe-campaign.md` and the end result would be the same.

It's not a problem but you could have a simpler structure here (unless you plan to add more docs in the `adobe-campaign` folder). Same applies to all 3 marketing docs.

Reply:
There's features slated for the near future which will probably require additional pages for each integration.